content/asset-tracking/index.textile

```diff
@@ -0,0 +1,165 @@
+---
+title: Asset Tracking
+section: asset-tracking
+index: 0
+meta_description: "The Ably Asset Tracking solution provides a way to track multiple assets in realtime."
+meta_keywords: "Ably, Asset Tracking SDK, Mapbox, Mapbox Navigation SDK, Mapbox location enhancement, Ably realtime, map matching, location tracking, Publishing SDK, Subscribing SDK"
+languages:
+  - javascript
+  - android
+  - objc
+  - java
+  - swift
+jump_to:
+  Help with:
+    - Battery optimization scenarios#battery-optimization
+    - Resolution#resolution
+    - Resolution set#resolution-set
+    - Resolution constraints#resolution-constraints
+    - Resolution policy#resolution-policy
+    - See also#see-also
+---
+
+The Ably Asset Tracking solution provides a way to track the location of multiple assets in realtime. An asset is any object that moves geographically, and whose position needs to be monitored. The Ably Asset Tracking solution is powered by the Ably realtime network, the Ably Asset Tracking SDKs, and the Mapbox Navigation SDK with location enhancement.
+
+The Ably Asset Tracking solution provides two SDKs:
+
+* Publishing SDK - for embedding in apps on the asset to be tracked.
+* Subscribing SDK - for embedding in apps that want to observe the asset being tracked.
+
+As Ably is used as the underlying transport, you have direct access to your data and can use "Ably Integrations":https://ably.com/integrations for a wide range of applications, in addition to direct realtime subscriptions. Examples include: 
+
+* Passing data to a third-party service.
```

Comment:
We need to capture the idea that it includes any realtime processing that's not via a realtime connection. It doesn't have to be a third-party.

Maybe "passing data to another service for realtime processing or tracking"